source/javascripts/components/_Trigger.js.erb

```diff
@@ -46,12 +46,32 @@
 			}
 		};
 
-		Trigger.prototype.workflowID = function(workflowID) {
-			if (workflowID !== undefined) {
-				this.triggerConfig.workflow = workflowID;
+		Trigger.prototype.target = function(target) {
+			if (!target) {
+				return this.targetType() + "#" + this.targetValue()
 			}
 
-			return this.triggerConfig.workflow;
+			var parts = target.split("#");
+			this.targetType(parts[0]);
+			this.targetValue(parts[1]);
+		}
+		
+		Trigger.prototype.targetValue = function(targetValue) {
```

Comment:
One more nitpicking, I'd suggest using **target ID** instead of **target value** everywhere.